Created action to handle PR's

colin-gourlay opened the pull request:
A basic implementation to demonstrate that the codebase is compiled and unit tests are run when a PR is made - as per #7

cgourlay committed:
Created action to handle PR's

A basic implementation to demonstrate that the codebase is compiled and unit tests are run when a PR is made - as per #7

cgourlay committed:
Bumped the version of .net

cgourlay committed:
Update pull request action

Temporarily ignore example projects

cgourlay committed:
Update pullrequest.yml

iamandylamb reviewed (commented):
I don't think you need the `dotnet remove` command at all. The incompatible projects have been excluded from the 'release' configuration, so when you `dotnet build --configuration Release` it'll only pick up the compatible projects.

cgourlay committed:
Update PR action

The build configuration in the solution has been updated - so there is no need to remove the projects from the sln via a GitHub action

iamandylamb commented:
I think you need to specify the configuration for the tests too. `dotnet test --configuration Release`.

cgourlay committed:
Merge pull request #10 from iamandylamb/master

Update feature branch

cgourlay committed:
Update pullrequest.yml

cgourlay committed:
Update pullrequest.yml

colin-gourlay commented:
Got there in the end! Thanks @iamandylamb
